rampwf/utils/testing.py

```diff
@@ -54,32 +62,18 @@ def assert_submission(ramp_kit_dir='./'):
             score_type.name, round(mean, score_type.precision),
             round(std, score_type.precision + 1)))
 
-
-def assert_backend(ramp_kit_dir='./'):
-    """Helper to test the backend from a ramp-kit.
-
-    Parameters
-    ----------
-    ramp_kit_dir : str,
-        The directory of the ramp-kit to be tested for submission.
-
-    Returns
-    -------
-    None
-
-    """
-    problem = imp.load_source('', join(ramp_kit_dir, 'backend.py'))
-    print('Preparing {} data'.format(problem.problem_title))
-    problem.prepare_data()
-    print('Testing {} backend'.format(problem.problem_title))
-    print('Reading file ...')
-    X_train, y_train = problem.get_train_data(path=ramp_kit_dir)
-    X_test, y_test = problem.get_test_data(path=ramp_kit_dir)
+    problem = imp.load_source('', join(ramp_kit_dir, 'problem.py'))
+    print('Testing {}'.format(problem.problem_title))
+    print('Reading train and test files from {}/data ...'.format(
+        ramp_data_dir))
+    X_train, y_train = problem.get_train_data(path=ramp_data_dir)
+    X_test, y_test = problem.get_test_data(path=ramp_data_dir)
     prediction_labels = problem.prediction_labels
     score_types = problem.score_types
-    print('Training model ...')
+    print('Training {}/submissions/{} ...'.format(ramp_kit_dir,
+                                                  submission_name))
     cv = list(problem.get_cv(X_train, y_train))
-    module_path = join(ramp_kit_dir, 'submissions', 'starting_kit')
+    module_path = join(ramp_kit_dir, 'submissions', submission_name)
     train_train_scoress = np.empty((len(cv), len(score_types)))
```

Comment:
is there a reason behind the double "S" at the end of scores throughout the file ?